GliaWidgets/SecureConversations/SecureConversations.SendMessageButton.swift

```diff
@@ -0,0 +1,158 @@
+import UIKit
+
+extension SecureConversations {
+    class SendMessageButton: UIControl {
```

Comment:
Cool, so we can use this same button in confirmation screen?

Reply:
Sure.